tests/conftest.py

```diff
@@ -107,11 +107,18 @@ def git_clone_fixtures(url: str, commit_hash: str, location: str) -> None:
 
 def pytest_sessionstart(session: Session) -> None:
 
-    fixtures_location = "tests"
+    fixtures_location = "tests/fixtures"
+
+    if not os.path.exists(fixtures_location):
+        os.mkdir(fixtures_location)
 
     for idx, props in test_fixtures.items():
         fixture_path = f"{fixtures_location}/{idx}"
 
+        # Set the path of the various downloaded fixtures
+        # as environment variables
+        os.environ[idx.upper()] = fixture_path
```

Comment:
Hm, is there no pytest native way to get this information into the tests? Using an environment variable seems a bit strange.

Reply:
Pytest generally discourages using global variables but the way our tests are set up, we need to use them if we want to avoid hardcoding the paths imo. I think in our scenario there aren't many downsides to using these custom global variables. 

As far as using a pytest native way for this is concerned, [this](https://github.com/pytest-dev/pytest/issues/3118#issuecomment-357702092) is the closest solution I could find but I am not sure it is much better than the `os.environ` variant :)